flow.css

```
@@ -0,0 +1,490 @@
+/* Flow UI - CSS component Library by Renuka Paturkar */
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@500&display=swap");
+
+:root {
+  --primary-color: #5b21b6;
+  --white-color: #f9fafb;
+  --red-color: #dc2626;
+  --green-color: #059669;
+  --blue-color: #2563eb;
+  --lightblue-color: #3b82f6;
```

```suggestion
  --light-blue-color: #3b82f6;
```
You can use this naming or camelCase one. It's easier to differentiate two words.

---

Hi Nitesh, Thank you so much for such a great review, I get to learn a lot while making  these changes, I have made the changes you suggested , Please let me know if they are okay, Thank you once again for review.